public/css/pages/index.css

```diff
@@ -127,7 +127,7 @@ main {
     cursor: pointer;
 }
 
-#table-filter-apply {
+#table-filter-apply, #table-filter-clear {
```

Comment:
Use class `.table-filter` instead of id for general styling.

Reply:
refer to [bade27d](https://github.com/erik-lance/TitafindsXG5/pull/11/commits/bade27df50803c2d1b36867abf1f53d09f8dae21)